src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            Files.deleteIfExists(Path.of(toFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can not delete if exist file " + e);
+        }
+        List<List<String>> records = readCsvToList(fromFileName);
+        int[] totalSupplyBuyResult = new int[3];// index 0 - supply, 1 - buy, 2 - result
+        calculateSupplyAndBuy(totalSupplyBuyResult, records);
+        totalSupplyBuyResult[2] = totalSupplyBuyResult[0] - totalSupplyBuyResult[1];
+        String[] resultData = new String[3];
+        resultData[0] = "supply," + totalSupplyBuyResult[0];
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)